client/src/components/CardExtraLarge.js

```diff
@@ -45,7 +41,7 @@ const CardExtraLarge = ({ title, imgSrc, description, price, service }) => {
       <article>
         <h3>Leistungsumfang:</h3>
         <ul>
-          {service.map((item) => {
+          {[service].map((item) => {
```

Comment:
I think it is not necessary to use the square bracket here. In your story service is an array, so it should work without the square brackets?!